workers/ohsome_quality_analyst/api.py

```diff
@@ -211,6 +217,42 @@ async def _fetch_report(
     return response
 
 
-@app.get("/regions")
+@app.get("/list_regions")
 async def get_available_regions():
+    """List names of available regions."""
     return await db_client.get_available_regions()
+
+
+@app.get("/list_indicators")
+async def list_indicators():
```

Comment:
As this is only returning the names of the indicators (in contrast to `/list_regions`), wouldn't it be better to name it to something like `/list_indicator_names`? // cc @matthiasschaub 

The same question applies to the other new endpoints.

Reply:
Sounds reasonable to me. In this case I would change the commands of the CLI as well. On the other hand this leads to quite long commands with are tedious to write.

I would prefer shorter names with good description. The docstring of the function will also be displayed in the swagger documentation. Maybe this will be enough to avoid confusion. E.g. on the CLI side this looks like following:

```python
@cli.command("list-indicators")
def list_indicators():
    """List available indicators and their metadata."""
```

```bash
oqt --help
[...]
Commands:
  list-indicators        List available indicators and their metadata.
```

But I am open for changing the command/enpoint name.

Reply:
Regarding the format. It should be formatted in lower camel case like the other parameters of the API. E.g. `listIndicators`. I find that hard to read and would prefer `list-indicators` as  this [article](https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/) suggests but this would be a change for another PR and needs to be discussed. Right now consistency is more important.